src/views/index.html

```diff
@@ -31,7 +31,7 @@ <h2 id="splash-hack-cambridge-wordmark">Hack<br>Cambridge<br><span class="pixel"
                             <div class="five unit column">
                                 <div id="splash-intro-text">
                                     <p>
-                                        In January 2018, Cambridge University’s annual hackathon will yet again bring together 300 top Computer Scientists and Engineers from the best universities worldwide.<br>For 24 hours they will build, break and innovate to produce projects that push the boundaries of technology.
+                                        In January 2018, Cambridge University’s annual hackathon will yet again bring together 300 Hackers, Programmers, Computer Scientists, Engineers, and more, from universities all over the world.<br>For 24 hours they will build, break and innovate to produce projects that push the boundaries of technology.
```

Comment:
Two thoughts:

- I think capitalising these names looks a bit strange
- "Engineers" and "Computer Scientists", I believe, are not universally understood as "someone who studies engineering or computer science". Maybe leave them out? e.g. "hackers, programmers, designers and more"

Reply:
Good shout, changing now